stream_alert/alert_processor/outputs.py

```diff
@@ -353,3 +353,61 @@ def dispatch(self, **kwargs):
         )
 
         self._log_status(resp)
+
+@output
+class LambdaOutput(AWSOutput):
+    """LambdaOutput handles all alert dispatching for AWS Lambda"""
+    __service__ = 'aws-lambda'
+
+    def get_user_defined_properties(self):
+        """Get properties that must be asssigned by the user when configuring a new S3
```

Comment:
I have no idea what this paragraph means 😆  what sensitive or unique information? what properties? when/why would someone need to do this? too vague